bin/generate-tiles

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGHOST="${POSTGRES_HOST:-${PGHOST?}}" # "xxx.xxx.xxx.xxx&host=xxx.xxx.xxx.xxx&host=..."
+export PGDATABASE="${POSTGRES_DB:-${PGDATABASE?}}"
+export PGUSER="${POSTGRES_USER:-${PGUSER?}}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-${PGPASSWORD?}}"
+export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
+
+export FUNCZXY="${FUNCZXY:-getmvt}"
+export CPU=${CPU:-4}  # number of CPUs per postgres server
```

Comment:
I suggest `COPY_CONCURRENCY` instead of `CPU`.
https://github.com/openmaptiles/openmaptiles/blob/master/.env#L49

Reply:
I used `COPY_CONCURRENCY` instead of `CPU`. I don't know what should I set for default value though. Set `1` for now.